index.html

```diff
@@ -166,13 +166,21 @@ <h2>
           <dd>
             <p>Prompts the user for permission to live-capture their display.</p>
             <p>
-              The user agent MUST let the end-user choose which display surface to share
-              out of all available choices every time, and MUST NOT use
-              <var>constraints</var> to limit that choice. Instead,
-              <var>constraints</var> MUST be applied to the media chosen by the user,
-              only after they have made their selection. This prevents an application
-              from influencing the selection of sources, see <a href="#constraints"></a>
-              for details.
+              The user agent MUST let the end-user choose which [=display surface=] to share
+              out of all available choices every time, and MUST NOT use <var>constraints</var>
+              to limit that choice.
+            </p>
+            <p>
+              The user agent MAY allow the application to use the {{displaySurface}} constraint to
+              signal a preference for a specific [=display surface=] type. The user agent MUST still
+              offer the user unlimited choice of any [=display surface=], but MAY order the sources
+              offered to the user according to this constraint.
```

Comment:
I suggest a minor rewording to clarify the MAY applies to the user agent not the application, and to not state things twice:  
```suggestion
              The user agent MAY use the presence of the {{displaySurface}} constraint and its value to
              influence the presentation to the user of the sources to pick from. The user agent MUST still
              offer the user unlimited choice of any [=display surface=].
```